### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by chambliss**

### CODE REVIEW

File: processing/combine_en_data.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        new.writelines(ee_data)
+
+    # Print success message
+    print('Combined {} and saved new dataset to {}.'
+         .format(dataset_path_list, combined_path))
+
+    return None
+
+
+if __name__ == '__main__':
```
Comment (ninalopatina) [inline processing/combine_en_data.py]: You'll want to put main.py as a separate file, you can take a look at this for reference: https://github.com/ninalopatina/Rocket_RL/blob/master/python/main.py

File: processing/combine_en_data.py [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+    conll_path = '../data/en/CONLL2003/'
```
Comment (ninalopatina) [inline processing/combine_en_data.py]: paths and other variables you define should go in a config.yml file such as https://github.com/ninalopatina/Rocket_RL/blob/master/config/config.yml

File: processing/combine_factRu2016.py [hunk 3]
```diff
@@ -0,0 +1,315 @@
...
+    label_df['lag(n_tokens)'] = label_df['n_tokens'].shift(1)
+
+    cond1 = (label_df['n_tokens'] == 1) & (label_df['lag(n_tokens)'].isna())
+    label_df['label1'] = np.where(cond1, 'B-' + label_df['label'], np.nan)
+
+    cond2 = (label_df['n_tokens'] >= 1) & (label_df['lag(n_tokens)'].isna())
+    label_df['label2'] = np.where(cond2, 'B-' + label_df['label'], np.nan)
+
+    # Label and lag(n_tokens) are both non-null (indicates being inside of a phrase)
+    cond3 = (label_df['label'].notna()) & (label_df['lag(n_tokens)'].notna())
```
Comment (ninalopatina) [inline processing/combine_factRu2016.py]: Is there output for how many are in each condition?

Comment (chambliss) [reply processing/combine_factRu2016.py]: clarification: how many rows fall into cond1, cond2, cond3

**Pull request merged**